Clinical-Genomics/cgweb
# Require elution buffer for mip exome and panel applications in Orderportal

This PR adds the requirement of elution buffer for mip exome and panel applications 

Contact: @annagellerbring 

How to test:
1. install branch on clinical-db
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA or EX
1. try to save sample

Expected outcome:
Red box around elution buffer

How to test submission of elution buffer:
1. install branch on clinical-db
1. create mip order in op
1. add family
1. add sample
1. select any application starting with PA or EX
1. select elution buffer
1. submit
1. open sample in LIMS

Expected outcome:
Elution buffer on the LIMS sample

- [ ] Code reviewed by 
- [ ] Tested by 
- [ ] Deploy OK:d by 

This is a minor **version bump** because it changes the UI in a backwards compatible manner
